proto/store/risk.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+import "google/api/expr/v1alpha1/syntax.proto";
```

Comment:
https://github.com/googleapis/googleapis/tree/master/google/api/expr/v1beta1

Use v1beta1 if possible.

Reply:
v1alpha1 is actually the one used in cel-go and is more recently updated.